### PR TITLE
feat: add debugtron

### PR DESCRIPTION
**Pull request by pd4d10**
[Debugtron](https://github.com/bytedance/debugtron) is a tool for in-production Electron based app debugging.

Screenshot

### CODE REVIEW

**Comment by HashimotoYT** [comment]
@pd4d10 As temporary solution you can add your app at the end of this file https://github.com/electron/apps/blob/master/lib/grandfathered-descriptions.js with comment why he was added.

**Comment by pd4d10** [comment]
@HashimotoYT  Thanks! Commit updated.

It is because the word "Electron" is necessary to describe the feature of Debugtron.

File: lib/grandfathered-descriptions.js [hunk 1]
```diff
@@ -130,6 +130,7 @@ module.exports = [
   'data-store',
   'datazenit',
   'dbglass',
+  'debugtron',
```
Comment (HashimotoYT) [inline lib/grandfathered-descriptions.js]: Please add comment why he was added. This case is an exception than default action.

Comment (pd4d10) [reply lib/grandfathered-descriptions.js]: OK. Comment added

**Pull request merged**